addons/custom-menu-bar/menu-labels.css

```diff
@@ -0,0 +1,18 @@
+[class*="menu-bar_file-group"] > [class*="menu-bar_hoverable"] > img {
+  display: none;
+}
+
+[class*="menu-bar_hoverable"] {
+  justify-content: center;
+}
+
+[class*="menu-bar_hoverable"] > span {
+  margin-right: 0;
+  padding-right: 0;
+}
```

Comment:
Looks like there's the tiniest bit of padding/margin to the left of the labels - you can see while hovering over them.
![Labels with leftover space to the left](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/42471653-7921-4ba5-acbc-be2a1b0b36c0)

This can probably be fixed by changing both margin right and margin left or simply removing it from all sides with `margin: 0` and `padding: 0`.

(In that case, you might also want to move that section into `@media (max-width: 1024px) {}` so it doesn't apply to the icons.)